pupil_src/launchables/eye.py

```diff
@@ -655,6 +685,17 @@ def window_should_update():
                         )
                     except KeyError as err:
                         logger.error(f"Attempt to load unknown plugin: {err}")
+                elif (
+                    subject.startswith("stop_eye_plugin")
+                    and notification["target"] == g_pool.process
+                ):
+                    try:
+                        plugin_to_stop = g_pool.plugin_by_name[notification["name"]]
+                    except KeyError as err:
+                        logger.error(f"Attempt to load unknown plugin: {err}")
+                    else:
+                        plugin_to_stop.alive = False
+                        g_pool.plugins.clean()
```

Comment:
Please extract this part into a separate PR as a solution for #2035

Reply:
Opened PR https://github.com/pupil-labs/pupil/pull/2039